src/components/transactions/TxDetails/Summary/TxDataRow/styles.module.css

```diff
@@ -11,6 +11,12 @@
   word-break: break-all;
 }
 
+.detailstitle {
+  font-weight: 400;
+  word-break: break-all;
+  color: var(--color-text-secondary);
```

Comment:
Adding a new class is not necessary. The `color` can be appended to `title` as it is only used in `TxDataRow`.